lib/code_corps/policy/stripe_connect_account.ex

```diff
@@ -3,11 +3,11 @@ defmodule CodeCorps.Policy.StripeConnectAccount do
 
   alias CodeCorps.{StripeConnectAccount, User}
 
-  def show?(%User{} = user, %StripeConnectAccount{} = stripe_connect_account),
-    do: stripe_connect_account |> get_organization() |> owned_by?(user)
+  def show?(%User{} = user, params),
+    do: params |> get_organization() |> owned_by?(user)
```

Comment:
This was an unnecessary change here. `show?` should still accept an account as a second argument.